src/linux/backend-v4l2.cpp

```diff
@@ -789,8 +805,9 @@ namespace librealsense
             // assign unique id for mipi by appending camera id to bus_info (bus_info is same for each mipi port)
             // Note - jetson can use only bus_info, as card is different for each sensor and metadata node.
             info.unique_id = bus_info + "-" + std::to_string(cam_id); // use bus_info as per camera unique id for mipi
+            info.dfu_device_path = "/dev/d4xx-dfu504"; // Use legacy DFU device node used in firmware_update_manager
```

Comment:
Does this hard coded path say we dont support multi camera DFU?

Reply:
it comes form fw-update-helper.cpp